.gitattributes

```diff
@@ -30,3 +29,7 @@ corporate/tests/stripe_fixtures/*.json -diff
 *.bmp binary
 *.mp3 binary
 *.pdf binary
+
+# Treat SVG files as binary to avoid newline conversion, but use
+# `cat` to diff on code changes.
+*.svg diff.cat.binary
```

Comment:
I don’t think we need to avoid newline conversion in SVGs? We have 7 SVGs with DOS newlines; we can just convert them and omit this special case.

Reply:
Great, thank you @andersk. I've pushed with a simpler diff-only line, and included a commit that does the CRLF -> LF normalization on those 7 SVGs (none of which are in the icon pipeline, of course, but good to have those addressed anyway).